fullscreen.bs

```diff
@@ -11,6 +11,11 @@ Markup Shorthands: css no
 </pre>
 
 <pre class=link-defaults>
+spec:html
+    type:dfn; text:allowed to use
+    type:dfn; text:showing hint popover list
+    type:dfn; text:showing auto popover list
```

Comment:
These aren't currently exported: PR to fix that https://github.com/whatwg/html/pull/11140

Reply:
Probably just remove these from this PR then, right?